lib/transaction_repository.rb

```diff
@@ -1,16 +1,18 @@
-require 'csv'
+require_relative 'file_loader'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.